src/deltaBalances/redux/contractFunctionMiddleware.js

```diff
@@ -7,13 +7,33 @@ export default function deltaBalancesMiddleware(store) {
       case 'GET_DELTA_BALANCES_ALL_BALANCES_FOR_MANY_ACCOUNTS':
         contractFunctions
           .getDeltaBalancesAllBalancesForManyAccounts(action.users, action.tokens)
-          .then(action.resolve)
+          .then(response => {
+            store.dispatch({
+              type: 'GOT_CALL_RESPONSE',
+              response: response && response.toString ? response.toString() : response,
```

Comment:
if `typeof response === object` this will resolve to `"[object Object]"`

Reply:
@perich I'm not 100% on what could be returned by this function, or what smart contracts are capable of returning. So I just thought better safe than sorry (maybe it could return `null`?)